engine/lib/phx/src/engine/engine.rs

```diff
@@ -483,4 +521,276 @@ impl Engine {
         Metric::reset();
         Profiler::end();
     }
+
+    // === Render Thread Control ===
+
+    /// Start the multithreaded render system.
+    ///
+    /// This transfers the GL context to a dedicated render thread.
+    /// All GL operations will then go through command mode.
+    /// Returns true if successfully started.
+    #[bind(name = "StartRenderThread")]
```

Comment:
Here and below `#[bind(name =  ...)]` is not needed. Procedural macro will generate this name automatically.
This attribute is needed only when you want a completely different name on the Lua side:
```rust
#[bind(name = "Create")]
pub fn new() -> Self { ... }
```